app/main.py

```diff
@@ -1,3 +1,41 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.data import Data
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        Data.data = json.load(file)
+
+    Customer.generate_unit()
+    Shop.generate_shop()
+
+    for customer in Customer.group_customers:
+        print(f"{customer.name} has {customer.money} dollars")
+
+        shop_choice = [None, float("inf")]
```

Comment:
Why you need a list here?
You just use it everywhere like two separate components